Archon.Http.Tests/FakeHttpHandler.cs

```diff
@@ -3,21 +3,17 @@
 using System.Net.Http;
 using System.Threading;
 using System.Threading.Tasks;
-using System.Web.Http;
-using System.Web.Http.Hosting;
 
-namespace Archon.WebApi
+namespace Archon.Http.Tests
 {
-	public class FakeHttpHandler : HttpMessageHandler
+	class FakeHttpHandler : HttpMessageHandler
```

Comment:
hmm yeah, I [removed that over a year ago](https://github.com/civicsource/http/pull/2/commits/e5423d181c78930a8eda3ba594dfd7e0e3ad391a).

I don't think we are quite ready to drop it just yet. Let me put it back.